benchmarks/gbench/common/distributed_vector.cpp

```diff
@@ -256,3 +256,18 @@ static void Mul_Serial(benchmark::State &state) {
 }
 
 DR_BENCHMARK(Mul_Serial);
+
+static void Inclusive_Scan(benchmark::State &state) {
```

Comment:
For comparison, please add a benchmark for a std::inclusive_scan with a parallel execution policy

Reply:
added, when running on 1 process on my local GPU, there is no difference
```
./mhp-quick-bench --benchmark_filter=Inclusive_Scan*
-----------------------------------------------------------------------------------------
Benchmark                                Time             CPU   Iterations UserCounters...
------------------------------------------------------------------------------------------
Inclusive_Scan/real_time               131 ms          131 ms            5 bytes_per_second=11.4077G/s footprint=1.49012G
Inclusive_Scan_Serial/real_time        131 ms          131 ms            5 bytes_per_second=11.3972G/s footprint=1.49012G
```